EIPS/eip-4886.md

```diff
@@ -0,0 +1,120 @@
+
+Mapping between an address and its Nominator and Delivery address is a simple process as shown below:
+
+    Contract / Dapp                        Register
+
+      |                                       |
+      |------------- 0x4567..---------------> |
+      |                                       |
+      | <-------nominator: 0x1234..---------- |
+      |         delivery: 0x9876..            |
+      |                                       |
+
+The protocol is fully backwards compatible. If it is passed an address that does not have an active mapping it will pass back the received address as both the Nominator and Delivery address, thereby preserving functionality as the address is acting on its own behalf.
+
+    Contract / Dapp                        Register
+
+      |                                       |
+      |------------- 0x0222..---------------> |
+      |                                       |
+      | <-------nominator: 0x0222..---------- |
+      |         delivery: 0x0222..            |
+      |                                       |
+
+If the EPS register is passed the address of a Nominator it will revert. This is of vital importance. The purpose of the proxy is that the Proxy address is operating on behalf of the Nominator. The Proxy address therefore can derive the same benefits as the Nominator (for example discord roles based on the Nominator's holdings, or mint NFTs that require another NFT to be held). It is therefore imperative that the Nominator in an active proxy cannot also interact and derive these benefits, otherwise two addresses represent the same holding. A Nominator can of course delete the Proxy Record at any time and interact on it's own behalf, with the Proxy address instantly losing any benefits associated with the proxy relationship.
+
+## Rationale
+
+The rationale for this design was to provide an easy and convenient way to prove ownership of assets while putting those assets at absolutely zero risk of loss. The motivation was seeing a number of attacks making use of the fact that users need to interact with a wallet holding assets in order to derive benefits of that asset ownership.
+
+In addition to the loss of user's assets it is clear that these events damage confidence in the community in general. Users make decisions based on both actual and perceived risk, and it presents a barrier for new users accessing and using Ethereum.
+
+Our vision is an Ethereum where users setup a new hardware wallet for assets they wish to hold long-term, then make one single contract interaction with that wallet: to nominate a hot wallet proxy. That user can always prove they own assets on that address, and they can specify it as a delivery address for new asset delivery.
```

Comment:
The rationale section should give the reasons behind any specific decisions within the EIP itself, and not reasons for the EIP as a whole. This section reads more like "Motivation".

Reply:
Yes, true! I have changed this section now. Thanks.